installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="installing-ibm-cloud-restricted"]
+= Installing a cluster on IBM Cloud VPC in a restricted network
+include::_attributes/common-attributes.adoc[]
+:context: installing-ibm-cloud-restricted
+
+toc::[]
+
+In {product-title} {product-version}, you can install a cluster in a restricted network by creating an internal mirror of the installation release content that is accessible to an existing Virtual Private Cloud (VPC) on {ibm-cloud-name}.
```

Comment:
Thanks again for this, Jeff. As discussed, rather than add it to the abstract of the topic, I updated About installations in restricted networks to address each of the points you highlighted.